extensions/ql-vscode/src/variant-analysis/shared/variant-analysis.ts

```diff
@@ -287,9 +287,11 @@ export function getSkippedRepoCount(
     return 0;
   }
 
-  return Object.values(skippedRepos).reduce(
-    (acc, group) => acc + group.repositoryCount,
-    0,
+  return (
+    (skippedRepos.accessMismatchRepos?.repositoryCount ?? 0) +
+    (skippedRepos.notFoundRepos?.repositoryCount ?? 0) +
+    (skippedRepos.noCodeqlDbRepos?.repositoryCount ?? 0) +
+    (skippedRepos.overLimitRepos?.repositoryCount ?? 0)
```

Comment:
I think this can be fixed while retaining the same method by using some type parameters:


```suggestion
  return Object.values(skippedRepos).reduce<number>(
    (acc, group: VariantAnalysisSkippedRepositoryGroup) =>
      acc + group.repositoryCount,
    0,
```